renovate.json

```diff
@@ -78,6 +78,13 @@
     {
       "groupName": "typescript",
       "matchPackageNames": ["typescript", "prettier", "@types/jsdom"],
+      "reviewers": ["team:kibana-operations"],
+      "matchBaseBranches": ["main"],
+      "labels": ["Team:Operations", "release_note:skip"],
+      "enabled": true
+    },
+    {
+      "groupName": "typescript-eslint",
```

Comment:
typescript minor versions typically come with breaking changes, typescript-eslint has been stable.  Splitting this up to prevent delays.